### ISSUE TITLE
In case of failure, duplicate emails are sent

### ISSUE THREAD

**Issue by robersonfaria** [robersonfaria/laravel-database-schedule#17]
When selecting to send email on success and failure, when a command fails, duplicate email is sent. I still haven't found a solution to the problem, apparently it's a behavior of the framework.

Case also interests @PedroPMS

**Comment by PedroPMS** [comment]
The email output options doesn't provide a method to send the output only in case of succes.

`emailOutputTo` -> send the output regardless of the result

`emailOutputOnFailureTo` -> send output only on failure

I suggested a new method in laravel github the handle this case:

`emailOutputOnSuccessTo` -> send output only on success

[Issue](https://github.com/laravel/framework/issues/37735)

**Comment by robersonfaria** [comment]
That's right, I was studying some way to create a new method or overwrite another one. But so far I haven't found a way.

Thanks for the open issue.
